src/sentry/api/serializers/models/integration.py

```diff
@@ -152,12 +152,15 @@ def get_attrs(self, item_list, user, *args, **kwargs):
         ints_by_id = {i.id: i for i in item_list}
         for ei in external_issues:
             # TODO(jess): move into an external issue serializer?
+            installation = ints_by_id[ei.integration_id].get_installation(
+                self.group.organization.id)
```

Comment:
Any chance of a KeyError here?

Reply:
It doesn't appear so. If you take a look at line 148, the `external_issues` are a list that have `integration_ids` in the `item_list` that's used to make the `ints_by_id` dict.